styled/resources.js

```diff
@@ -0,0 +1,72 @@
+import styled from 'styled-components/native';
+import { Subhead, Title, Body } from '../components/BaseComponents';
+
+export const Card = styled.View`
+  border-bottom-width: 1px;
+  border-color: #ebebeb;
+  padding: 10px;
```

Comment:
Make sure this is precise. Use `padding: 12px 24px;` for 12px top bottom, 24 px left and right to match Figma.